doc/releases/changelog-dev.md

```diff
@@ -26,6 +37,11 @@
 
 <h3>Bug fixes</h3>
 
+* The operation `OrbitalRotation` previously was wrongfully registered to satisfy
+  the four-term parameter shift rule, it now will be decomposed instead when
+  using the parameter-shift rule.
+  [(#2180)](https://github.com/PennyLaneAI/pennylane/pull/2180)
```

Comment:
Quick question, rather than forcing the `OrbitalRotation` gate into a decomposition, could we just do?

```python
@property
def grad_recipe(self):
    coeffs, shifts = qml.gradients.generate_shift_rule(self.parameter_frequencies[0])
    return np.stack([coeffs, np.ones_like(coeffs), shifts]).T
```

Reply:
Basically, just a temporary hotfix to have the `OrbitalRotation` use the 8-term shift rule directly?

Reply:
Funnily enough, the [known decomposition](https://github.com/PennyLaneAI/pennylane/blob/d09598c6e91424569df007ea51f0a097d806c40f/pennylane/ops/qubit/qchem_ops.py#L636-L652) has 4 parametrized gates, so there is no savings re: number of quantum executions

Reply:
Ah, yes, that definitely is a better option :)
I'd say the standard case for gates is that the number of decomposition shifts and generalized shifts exactly matches. :)

Reply:
Yeah, I was just hoping that maybe the known decomposition was poor 😆